tests/testthat/test-col_plan.R

```diff
@@ -186,6 +213,19 @@ test_that("From col plan spanning structures, get df to add to data",{
     )
   )
 
+  one_span_wide_data_rename <- apply_span_structures_to_data(tfrmt_obj = tfrmt_obj_one_span_rename, x = input_data)
+
+  expect_equal(
+    one_span_wide_data_rename,
+    tibble(
+      `__tlang_span_structure_column__1` = c(NA, NA, "test label", rep(NA, 7)),
+      columns = paste0("col",1:10),
```

Comment:
is col3 supposed to be renamed here?

Reply:
Renaming of values does not occur until select_col_plan. Here we are just trying to make the data format the same between span_strcture and multiple column formats